draco/analysis/delay.py

```diff
@@ -718,6 +718,212 @@ def process(self, ss: FreqContainerType) -> containers.DelaySpectrum:
         return delay_spec
 
 
+class DelaySpectrumWienerBase(task.SingleTask):
+    """Calculate the delay spectrum for each RA and DEC of any container with a frequency axis.
```

Comment:
This description isn't quite correct: in principle this task could operate on visibilities, in which case the input container would not have a dec axis